src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package Duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A Task that can be added to the Task List
+ * which has a deadline date after the description.
```

Comment:
Public classes must have `@author` and `@version` information in the JavaDoc comment. So something like this:

```java
/**
 *
 * A Task that can be added to the Task List
 * which has a deadline date after the description.
 *
 * @author Janice Chen
 * @version CS2103 AY21/22 Semester 1
 * 
*/
```